cdk/python/elasticloadbalancingv2-resources/elasticloadbalancingv2_resources/elasticloadbalancingv2_resources_stack.py

```diff
@@ -0,0 +1,200 @@
+from aws_cdk import Stack
+import aws_cdk as cdk
+import aws_cdk.aws_ec2 as ec2
+import aws_cdk.aws_elasticloadbalancingv2 as elasticloadbalancingv2
+from constructs import Construct
+
+class Elasticloadbalancingv2ResourcesStack(Stack):
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    # Applying default props
+    props = {
+      'cidrBlockForVpc': kwargs.get('cidrBlockForVpc', '186.0.0.0/24'),
+      'cidrBlockForSubnet1': kwargs.get('cidrBlockForSubnet1', '186.0.0.0/25'),
+      'cidrBlockForSubnet2': kwargs.get('cidrBlockForSubnet2', '186.0.0.128/25'),
+      'availabilityZoneForSubnet1': kwargs.get('availabilityZoneForSubnet1', 'us-east-1c'),
+      'availabilityZoneForSubnet2': kwargs.get('availabilityZoneForSubnet2', 'us-east-1b'),
+    }
+
+    # Resources
+    vpc = ec2.CfnVPC(self, 'VPC',
```

Comment:
Why are we using L1 constructs here?

Reply:
I tried adding using L1 constructs so that the stack is created with minimum number of resources. Will change them to L2 and push again.